Fix build in case of Java 17

Fix build if Java 17 installed

```
❯ ./gradlew             

FAILURE: Build failed with an exception.

* Where:
Settings file 'kaluga/settings.gradle.kts' line: 43

* What went wrong:
Could not open dsl generic class cache for script 'kaluga/gradle/ext.gradle' (~/.gradle/caches/7.0.2/scripts/9psnnhgac23u83vlibfy0er94).
> BUG! exception in phase 'semantic analysis' in source unit '_BuildScript_' Unsupported class file major version 61
```

--- a/example/ios/Supporting Files/gradle/wrapper/gradle-wrapper.properties
+++ b/example/ios/Supporting Files/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

--- a/gradle/publish.gradle
+++ b/gradle/publish.gradle
@@ -9,7 +9,7 @@ afterEvaluate {
                 release(MavenPublication) {
                     from components.release
 
-                    artifactId = module.name
+                    artifactId = name
                     groupId = "com.splendo.kaluga"
                     version = gradle.ext.library_version
                 }
@@ -17,13 +17,13 @@ afterEvaluate {
                 debug(MavenPublication) {
                     from components.debug
 
-                    artifactId = module.name
+                    artifactId = name
                     groupId = "com.splendo.kaluga"
                     version = gradle.ext.library_version
                 }
             } else {
                 kotlinMultiplatform { publication ->
-                    artifactId = module.name
+                    artifactId = name
                     groupId = "com.splendo.kaluga"
                     version = gradle.ext.library_version
                 }

--- a/gradle/publish.gradle
+++ b/gradle/publish.gradle
@@ -9,7 +9,7 @@ afterEvaluate {
                 release(MavenPublication) {
                     from components.release
 
-                    artifactId = name
+                    artifactId = project.name
                     groupId = "com.splendo.kaluga"
                     version = gradle.ext.library_version
                 }
@@ -17,13 +17,13 @@ afterEvaluate {
                 debug(MavenPublication) {
                     from components.debug
 
-                    artifactId = name
+                    artifactId = project.name
                     groupId = "com.splendo.kaluga"
                     version = gradle.ext.library_version
                 }
             } else {
                 kotlinMultiplatform { publication ->
-                    artifactId = name
+                    artifactId = project.name
                     groupId = "com.splendo.kaluga"
                     version = gradle.ext.library_version
                 }

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.2-all.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

